std_msgs/Header header
uint8 precision # 0: mm, 1: cm, 2: dm, 3: m
int16[] points # x y z
uint8[] intensities
